File: exs/mundo_1/java/007.java
import java.util.*;

class Main {
    public static void main(String[] args){
        Scanner x = new Scanner(System.in);
        System.out.print("Primeira nota: ");
        int nota1 = x.nextInt();
        System.out.print("Segunda nota: ");
        int nota2 = x.nextInt();
        float media = (nota1 + nota2) / 2;
        System.out.printf("Média = %.1f\n", media);
    }
}

File: exs/mundo_1/golang/007.go
/*
	Desafio: 007
	Problema: Desenvolva um programa que leia 
	as duas notas de um aluno, calcule e 
	mostre a sua média.
*/

package main 
import (
	"fmt"
)

func main(){
	var first float64
	var second float64
	var calc float64
	fmt.Print("Informe a primeira nota: ")
	fmt.Scanln(&first)
	fmt.Print("Informe a segunda nota: ")
	fmt.Scanln(&second)
	calc = first + second / 2
	fmt.Print("Média: ", calc)
}